.github/helm-ct.yml

```diff
@@ -3,3 +3,6 @@ chart-dirs:
   - helm
 chart-repos:
   - bitnami=https://charts.bitnami.com/bitnami
+debug: true
```

Comment:
should this stay?

Reply:
this setup will duplicated across all SoftwareMill repositories / projects for which Helm Charts exist - currently around 30 SML applications (both internal and open-source) are deployed using Helm; so this is just for additional diagnostic info, it simply outputs additional info during the "Run chart-testing (lint)" (e.g. https://github.com/softwaremill/bootzooka/runs/1447485540?check_suite_focus=true) and "Run chart-testing (install)" (e.g. https://github.com/softwaremill/bootzooka/runs/1447487632?check_suite_focus=true) jobs; I can remove this setting now if you insist, but IMHO we should leave it until we have this implemented across several setups in case any changes are needed

Reply:
ok, I was wondering if it's not a left-over from dev